server/test/org/apache/cloudstack/network/lb/CertServiceTest.java

```diff
@@ -297,48 +297,48 @@ public void runUploadSslCertNoRootCert() throws IOException, IllegalAccessExcept
 
         Assume.assumeTrue(isOpenJdk() || isJCEInstalled());
 
-        String certFile = URLDecoder.decode(getClass().getResource("/certs/rsa_ca_signed.crt").getFile(),Charset.defaultCharset().name());
-        String keyFile = URLDecoder.decode(getClass().getResource("/certs/rsa_ca_signed.key").getFile(),Charset.defaultCharset().name());
-        String chainFile = URLDecoder.decode(getClass().getResource("/certs/non_root.crt").getFile(),Charset.defaultCharset().name());
+        final String certFile = URLDecoder.decode(getClass().getResource("/certs/rsa_ca_signed.crt").getFile(),Charset.defaultCharset().name());
+        final String keyFile = URLDecoder.decode(getClass().getResource("/certs/rsa_ca_signed.key").getFile(),Charset.defaultCharset().name());
+        final String chainFile = URLDecoder.decode(getClass().getResource("/certs/non_root.crt").getFile(),Charset.defaultCharset().name());
```

Comment:
Please see line notes for 154-155 as those comments apply to lines 300-302 as well.